### DISCUSSION TITLE
Global/System-wide hot key for "Show KeePass window"

### DISCUSSION THREAD

**Discussion by ce603** [keepassxreboot/keepassxc#11161] [Ideas]
![image](https://github.com/user-attachments/assets/6d3b24e8-6cab-4342-87c1-0c4ee642951c)

In the standard KeePass program, there is an option to pull up the KeePass database using a user-definable shortcut called "Show KeePass window". I would like to see this in the KeePassXC version as well. As a long time user of KeePass migrating to KeePass XC, I have several hundred entries that do not have URLs in the URL field, so KeePassXC doesn't recognize them and I need to bring up the KeePass XC window. It would be very helpful to have a shortcut to do that. I'm not sure why this feature was removed; the global auto-type selected entry shortcut still exists in XC, just not the one to bring the window up.

**Comment by droidmonkey** [comment]
It's planned

**Reply by PatrickTerlisten** [reply]
Any update on this? I'm really missing this feature in XC.

**Reply by lippling** [reply]
I also would like to have this feature. In 1Password I git the global shortcut and can start typing right away.
